infra/app/storage.bicep

```diff
@@ -0,0 +1,56 @@
+param name string
+param location string
+param blobContainerName string
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2021-09-01' = {
```

Comment:
You should be able to use infra/core for this.

Reply:
In the `storage.bicep` file, not only `StorageAccount` and the corresponding `BlobServices` are created, but also the `QueueServices` with `StorageAccount` as the parent is created. infra/core does not support this part.

Reply:
Okay, in this case, please update the /infra/core modules to support this in this project, and then we can do a PR into azure-dev repo later.

Reply:
Updated.